packages/web/src/utils/route.ts

```diff
@@ -302,6 +302,13 @@ export const staticRoutes = new Set([
   EMPTY_PAGE,
   SIGN_IN_PAGE,
   SIGN_UP_PAGE,
+  ...SIGN_ON_ALIASES,
+  SIGN_UP_EMAIL_PAGE,
+  SIGN_UP_PASSWORD_PAGE,
+  SIGN_UP_HANDLE_PAGE,
+  SIGN_UP_FINISH_PROFILE_PAGE,
+  SIGN_UP_GENRES_PAGE,
+  SIGN_UP_ARTISTS_PAGE,
   NOTIFICATION_PAGE,
```

Comment:
We also have these routes too:
![image](https://github.com/AudiusProject/audius-protocol/assets/6711655/ae80a46c-dee5-4a2a-9c03-5b6a35f00005)